addons/mrp_subcontracting_purchase/tests/test_mrp_subcontracting_purchase.py

```diff
@@ -901,3 +901,46 @@ def test_return_subcontracted_product_to_supplier_location(self):
         return_picking = self.env['stock.picking'].browse(return_picking_id)
         return_picking.button_validate()
         self.assertEqual(return_picking.state, 'done')
+
+    @freeze_time('2000-05-05')
+    def test_global_visibility_days_affect_lead_time(self):
+        """ Don't count global visibility days more than once, make sure a PO generated from
+        replenishment/orderpoint has a sensible planned reception date.
+        """
+        try:
+            self.env['sale.order']
+        except KeyError:
+            self.skipTest('This test requires the installation of the following modules: sale')
+
+        from json import loads
+
+        wh = self.env.user._get_default_warehouse_id()
+        self.env['ir.config_parameter'].sudo().set_param('stock.visibility_days', '365')
+        self.finished2.seller_ids = [Command.link(self.env['product.supplierinfo'].create({
+            'partner_id': self.subcontractor_partner1.id,
+            'product_id': self.finished2.id,
+            'delay': 0,
+        }).id)]
+        final_product = self.finished2
+        orderpoint = self.env['stock.warehouse.orderpoint'].create({'product_id': final_product.id})
+        sale_order = self.env['sale.order'].create({
+            'partner_id': self.env['res.partner'].create({'name': 'no1'}).id,
+            'order_line': [Command.create({
+                'product_id': final_product.id,
+                'product_uom_qty': 2,
+            })],
+        })
+        sale_order.action_confirm()
+        r = orderpoint.action_stock_replenishment_info()
+        repl_info = self.env[r['res_model']].browse(r['res_id'])
+        lead_days = loads(repl_info.json_lead_days)
+        res = [desc for desc in lead_days['lead_days_description'] if desc[0] == 'Global Visibility Days']
+        self.assertEqual(len(res), 1)
```

Comment:
Let's just assert on the key `'lead_days_date'`
(And just do `today + ...` so we avoid freezing time)